templates/maps/google-maps-with-search.php

```diff
@@ -120,5 +125,39 @@ function autoCompleteAddress(){
                 }
             });
         }
+
+        function reverseGeocode( latLng ) {
+            let data = {
+                location: {
+                    lat: latLng.lat(),
+                    lng: latLng.lng(),
+                }
+            };
+
+            geocoder.geocode( data )
+                .then( function ( { results: addresses } ) {
+                    let address = addresses[0].address_components
+                        .filter( component => ! component.types.includes( 'plus_code' ) )
+                        .map( component => component.long_name )
+                        .join( ', ' );
+                    $input_add.val( address );
+                } )
+                .catch( e => console.log( e ) );
+        }
+
+        function dokan_debounce_delay( callback, ms ) {
+            var timer = 0;
```

Comment:
you've use `let` above, but `var` here? why? You should keep the code consistency.
same for bottom `context` and `args` variable.